src/server.test.ts

```diff
@@ -0,0 +1,10 @@
+import request from "supertest";
+import app from "./server";
+
+describe("GET /", () => {
+  it("responds with JSON message 'Hello from server!'", async () => {
```

Comment:
what if, if you are not able to get a response? or fail? network error? 

excetion management please

Reply:
A try/catch block has been added to handle potential errors.